share/spice/time/content.handlebars

```diff
@@ -1,4 +1,4 @@
-<div class="location">Time in <b>{{placeName}}, {{country}}</b></div>
+<div class="location">Time in <b>{{placeName}}, {{country}}</b> &#124; UTC{{offset}}</div>
```

Comment:
@MrChrisW let's use  `<span class="sep"></span>` instad of the `&#124;` -- this will ensure it has the UI @abeyang has asked for